en_US/rate-limit/rate-limit.md

```diff
@@ -2,34 +2,84 @@
-- The maximum publishing rate of data is 1MB per second per client
+- The maximum rate of connection establishment on the listener is 1000 per second.
+- The listener can accept a maximum of 10,000 connections within 60 minutes.
+- The maximum publishing rate for messages is 1000 per second per client.
+- The listener allows a burst of up to 10,000 messages within a short period every 60 minutes.
+- The maximum publishing rate for data is 1MB per second per client.
+- The listener allows a burst of up to 100MB within a short period every 60 minutes.
+
+## Node-Level Limiters
+
+Limiters can also operate at the node level, limiting the speed of individual client connections to each EMQX node and the rate at which messages or data are published to the node. EMQX nodes use the following types of limiters to specify rate limits:
+
+| Type           | Dashboard UI             | Description                                                  | Post-Overload Behavior                                       |
+| -------------- | ------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| bytes_rate     | Data Publish Rate        | The amount of data (in bytes) sent by a single client to each EMQX node | When the limit is reached, EMQX will drop QoS 0 messages and reject QoS 1 and QoS 2 messages with a "Quota Exceeded" error (0x97). |
+| bytes_burst    | Data Publish Burst       | The burst amount of data allowed per client, based on the regular `data publish rate` | When the limit is reached, EMQX will drop QoS 0 messages and reject QoS 1 and QoS 2 messages with a "Quota Exceeded" error (0x97). |
+| messages_rate  | Message Publish Rate     | The rate at which a single client sends messages to each EMQX node | When the limit is reached, EMQX will drop QoS 0 messages and reject QoS 1 and QoS 2 messages with a "Quota Exceeded" error (0x97). |
+| messages_burst | Message Publish Burst    | The number of messages allowed to be sent per node in bursts, based on the regular `message publishing rate` | When the limit is reached, EMQX will drop QoS 0 messages and reject QoS 1 and QoS 2 messages with a "Quota Exceeded" error (0x97). |
+| max_conn_rate  | Maximum Connection Rate  | The rate at which new connections are accepted per node      | When the limit is reached, EMQX will pause processing connections in the Accept queue, delaying or rejecting new connections. |
+| max_conn_burst | Maximum Connection Burst | The maximum number of connections that a node can accept in bursts | Pause receiving new connections                              |
+
+### Configuring Node-Level Limiters
+
+You can configure rate limits for each listener on the **Management** -> **MQTT Configuration** page in the Dashboard.
+
+Alternatively, you can configure them through the configuration file. For example, you can configure the following in `emqx.conf`:
+
+```bash
+limiter {
```

Comment:
Node-level limiters are configured per zone, so the config is
```
mqtt {
   limiter {
   ...
   }
}
```
for the zone defaults and

```
zones {
   myzone {
      mqtt {
         limiter {
         }
      }
   }
}
```
for explicit zones